### DISCUSSION TITLE
Split module `name`

### DISCUSSION THREAD

**Discussion by pkuczynski** [faker-js/faker#1194] [Ideas]
Module `name` currently includes things not related to `name` like `gender` and `job*`.

Proposal:
* move `gender` to a `person` module. Why: this module could be later on extended with things like `height`, `eyes`, etc.

  ```js
  faker.person.sex()
  faker.person.gender()
  faker.person.height() // idea for the future
  faker.person.eyes() // idea for the future
  ```
* move `job*` to a dedicated module:

  ```js
  faker.job.area()
  faker.job.descriptor()
  faker.job.title()
  faker.job.type()
  ```

* simplify `name` module methods (optional)

  ```js
  faker.name.first() // instead of faker.name.firstName()
  faker.name.last() // instead of faker.name.lastName()
  faker.name.full() // instead of faker.name.fullName()
  faker.name.prefix()
  faker.name.suffix()
  ```

**Comment by ST-DDT** [comment]
I would move the firstName and lastName method to the person module as well effectively renaming the name module to person.
As for the job method, we could move them to company or as a standalone module.

**Reply by pkuczynski** [reply]
* I like them to be shorter `faker.name.first()` vs `faker.name.firstName()`
* `faker.person.prefix` does not really make sense (but we could rename it to `faker.person.title()` maybe?)
* `faker.person.suffix` does not really make sense (but we could rename it to `faker.person.nameSuffix()`, although it looks really long so I would prefer `faker.name.suffix()`)

**Reply by ST-DDT** [reply]
`first` and `last` sound like array accessors to get the first or last element in an array.
I have to check the prefix and suffix locale data before I can decide on them. These might only be helpers functions for the old fullName/findName method and as such might not need public methods at all.
